### PR TITLE
zh-cn: create the translation of "Firefox 127 release note"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

create the translation of "Firefox 127 release note"

### CODE REVIEW

File: files/zh-cn/mozilla/firefox/releases/127/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  CSS {{cssxref("letter-spacing")}} 属性现在会将指定的字母间距均匀分配到每个字符的两侧。这与当前的行为（当前行为主要是将间距添加到一侧）不同（[Firefox bug 1891446](https://bugzil.la/1891446)）。
+
+- **`calc()` 相对颜色的色彩通道支持**：`layout.css.relative-color-syntax.enabled`。
+
+  CSS [`calc()`](/zh-CN/docs/Web/CSS/calc) 函数现在可以以[相对颜色](/zh-CN/docs/Web/CSS/CSS_colors/Relative_colors#使用_math_函数)解析色彩通道。
+
+- **JavaScript `Float16Array` 类型化数组**：`javascript.options.experimental.float16array`。
+
+  现已支持 {{jsxref("Float16Array")}} 类型化数组，以及用于通过 {{jsxref("DataView")}} 读取和设置 `Float16Array` 值的 {{jsxref("DataView.prototype.getFloat16()")}} 和 {{jsxref("DataView.prototype.setFloat16()")}}，以及用于将数字舍入到 16 位的 {{jsxref("Math.f16round()")}} 静态方法。新类型对于与 GPU 共享数据特别有用，特别是在需要权衡精度和内存消耗的用例中。（[Firefox bug 1833647](https://bugzil.la/1833647)。）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/127/index.md:90]:
```suggestion
  现已支持 {{jsxref("Float16Array")}} 类型化数组，以及用于通过 {{jsxref("DataView")}} 读取和设置 `Float16Array` 值的 {{jsxref("DataView.prototype.getFloat16()")}} 和 {{jsxref("DataView.prototype.setFloat16()")}}，以及用于将数字舍入到 16 位的 {{jsxref("Math.f16round()")}} 静态方法。新类型对于与 GPU 共享数据特别有用，特别是在需要以精度换取内存消耗的用例中。（[Firefox bug 1833647](https://bugzil.la/1833647)。）
```

File: files/zh-cn/mozilla/firefox/releases/127/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  - [`Set.prototype.union()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/union)——返回一个含有当前集合以及给定集合中的所有元素的新集合。
+  - [`Set.prototype.difference()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/difference)——返回一个含有当前集合中存在而给定集合中不存在的元素的新集合。
+  - [`Set.prototype.symmetricDifference()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/symmetricDifference)——返回一个含有存在于当前集合或给定集合，但不同时存在于两者中的元素的新集合。
+  - [`Set.prototype.isSubsetOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/isSubsetOf)——返回一个指示当前集合中的所有元素是否都存在于给定集合中的布尔值。
+  - [`Set.prototype.isSupersetOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/isSupersetOf)——返回一个指示给定集合中的所有元素是否都存在于当前集合中的布尔值。
+  - [`Set.prototype.isDisjointFrom()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set/isDisjointFrom)——返回一个指示当前集合是否与给定集合没有任何元素相同的布尔值。
+
+### SVG
+
+- [`lh` 和 `rlh`](/zh-CN/docs/Learn/CSS/Building_blocks/Values_and_units#line_height_units) 行高单元首先在 CSS 中得到支持（[在 Firefox 120 中](/zh-CN/docs/Mozilla/Firefox/Releases/120#css)），现在也在 SVG 中得到了支持。它们可以在 CSS 属性值（`stroke-width: 0.5lh`）和 SVG 属性值（`stroke-width="0.5lh"`）中使用。（[Firefox bug 1892089](https://bugzil.la/1892089)）。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/firefox/releases/127/index.md:36]:
```suggestion
- [`lh` 和 `rlh`](/zh-CN/docs/Learn/CSS/Building_blocks/Values_and_units#line_height_units) 行高单位首先在 CSS 中得到支持（[在 Firefox 120 中](/zh-CN/docs/Mozilla/Firefox/Releases/120#css)），现在也在 SVG 中得到了支持。它们可以在 CSS 属性值（`stroke-width: 0.5lh`）和 SVG 属性值（`stroke-width="0.5lh"`）中使用。（[Firefox bug 1892089](https://bugzil.la/1892089)）。
```

File: files/zh-cn/mozilla/firefox/releases/127/index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## 为 Web 开发者带来的变化
+
+### HTML
+
+- 现在禁止在 `<base>` 元素的 [`href`](/zh-CN/docs/Web/HTML/Element/base#href) 属性中使用 `data:` 和 `javascript:` URL。（[Firefox bug 1850967](https://bugzil.la/1850967)）。
+
+### CSS
+
+- 现已支持在使用 [`conic-gradient()`](/zh-CN/docs/Web/CSS/gradient/conic-gradient)、[`linear-gradient()`](/zh-CN/docs/Web/CSS/gradient/linear-gradient) 和 [`radial-gradient()`](/zh-CN/docs/Web/CSS/gradient/radial-gradient) 函数创建渐变时，以及在使用 [`repeating-conic-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-conic-gradient)、[`repeating-linear-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-linear-gradient) 和 [`repeating-radial-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-radial-gradient) 函数创建重复渐变时，使用 [`<color-interpolation-filters>`](/zh-CN/docs/Web/CSS/color-interpolation-filters)。（[Firefox bug 1861363](https://bugzil.la/1861363)）。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/releases/127/index.md:20]:
```suggestion
- 现已支持在使用 [`conic-gradient()`](/zh-CN/docs/Web/CSS/gradient/conic-gradient)、[`linear-gradient()`](/zh-CN/docs/Web/CSS/gradient/linear-gradient) 和 [`radial-gradient()`](/zh-CN/docs/Web/CSS/gradient/radial-gradient) 函数创建渐变时，以及在使用 [`repeating-conic-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-conic-gradient)、[`repeating-linear-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-linear-gradient) 和 [`repeating-radial-gradient()`](/zh-CN/docs/Web/CSS/gradient/repeating-radial-gradient) 函数创建重复渐变时，使用 [`<color-interpolation-method>`](/zh-CN/docs/Web/CSS/color-interpolation-method)。（[Firefox bug 1861363](https://bugzil.la/1861363)）。
```

**Review by jasonren0403 — APPROVED** [review]
Almost LGTM, some minor nits

**Pull request merged**